ui/selectors/custom-gas.test.js

```diff
@@ -109,9 +109,13 @@ describe('custom-gas selectors', () => {
       expect(isCustomPriceExcessive(mockState)).toStrictEqual(true);
     });
     it('should return false gas.basicEstimates.price 0x28bed01600 (175) (checkSend=true)', () => {
+      // Inconsistent state from the selectors? ui/ducks/send/send.js3L1117
+      // Is there suppose to be a send[gas]?
```

Comment:
Check

Reply:
No, send[gas] should always be an object, if it sometimes isn't then that needs to be addressed. gasPrice should always be at send.gas.gasPrice